deploy/osd-devaccess/01-osd-devaccess.ClusterRole.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osd-devaccess
+rules:
+  # devaccess can get, list, and watch all resources
+  - apiGroups:
+      - "*"
+    resources:
+      - "*"
+    verbs:
+      - get
+      - list
+      - watch
+  # devaccess can get and list nodes
```

Comment:
Are all these get/list entries for other objects redundant if the first rule is "get/list/watch all"?

Reply:
Yep, thank you.